### ISSUE TITLE
Not compatible with Laravel 10

### ISSUE THREAD

**Issue by Simonjonah** [kingflamez/laravelrave#110]
Not compatible with laravel 10 please update the verion.

**Comment by EmmanuelObua** [comment]
True, not compatible, any update please @Simonjonah @masroore @emmanix2002 @xwiz @frizikk

**Comment by damms005** [comment]
You may find [my maintained version](https://github.com/damms005/laravel-flutterwave) useful, because I need it for [my payments package](https://github.com/damms005/laravel-multipay).

Enjoy! 🥂
